easybuild/easyblocks/d/deepspeed.py

```diff
@@ -0,0 +1,61 @@
+from easybuild.easyblocks.generic.pythonpackage import PythonPackage
```

Comment:
please include comment block on + you as the author

Reply:
So the one from PythonPackage with me at the end you mean?

Reply:
Eh, doesn't seem to be what you meant based on other derived configs. I'll just do what has been done in others.